lib/README.md

```diff
@@ -96,6 +96,7 @@ The following scripts can be executed manually or as part of a GitHub Action:
   - a copy of the common [`styles`](../styles) folder
   - a copy of the document-specific `*/images` folder, if this exists.
 - [`npm run pdf`](build-pdf.mjs) runs the PDF conversion and writes the PDF document into the [`docs/*`](../docs) folder.
+- [`npm run zip <yyyy-MM-dd>`](zip.js) creates ZIP files for OASIS public review of documents with the given publication date. The ZIP files are written into the [`zip`](../zip) folder.
```

Comment:
Why upper-case "MM"?

Reply:
MM stands for month, mm stands for minutes.

Reply:
Well, then why not YYYY-MM-DD, following https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format?

Which tool or standard uses yyyy-MM-dd?

Reply:
> Which tool or standard uses yyyy-MM-dd?

https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html